esp_sysview/src/include/esp_trace_freertos_impl.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
why this is a public header file? do  you expect the user to include it in the application code?